src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const price = 40;
+
+  if (days >= 7) {
+    return price * days - 50;
```

Comment:
Don't repeat yourself, let's create a separate variable for price * days

Reply:
```suggestion
    return cost - 50;
```